src/cljs/notifikator/views.cljs

```diff
@@ -1,8 +1,79 @@
 (ns notifikator.views
-  (:require [re-frame.core :as re-frame]
+  (:require [reagent.core :as reagent]
+            [re-frame.core :as re-frame]
             [notifikator.subs :as subs]
-            ))
+            [notifikator.events :as events]))
 
-(defn main-panel []
-  (let [name (re-frame/subscribe [::subs/name])]
-    [:div "Hello from " @name]))
+(defn close-button
+  [id]
+  [:div {:class "close"
+         :on-click #(re-frame.core/dispatch [::events/close-message id])}
+   "X"])
+
+(defn message
+  [ind {:keys [id title description class]}]
+  ^{:key (str "msg-" id)}
+  [:div {:class class}
+   [:div {:class "title"} title (close-button id)]
+   [:div {:class "description"} description]])
+
+(defn messages-container
+  "If there are messages in the app-state create a message component
+   for each of them. Otherwise return nil. Which is fine for Hiccup."
+  []
+  (when-let [messages (re-frame/subscribe [::subs/messages])]
```

Comment:
This `when-let`'s condition is always going to be true, cause `re-frame/subscribe` returns (something-like-) a reagent's `atom`.

Reply:
Also, even though this works ok, the reagent's "form 1, 2, 3 component" rules also apply here.

Reply:
Thanks! Did not know that - still learning re-frame :)